src/Sherpa/Article.swift

```diff
@@ -83,7 +85,8 @@ internal struct Article {
 		// Title and body
 		title = dictionary["title"] as? String ?? ""
 		body = dictionary["body"] as? String ?? ""
-		if title.isEmpty || body.isEmpty {
+        html = dictionary["html"] as? String ?? ""
+		if title.isEmpty || ( body.isEmpty && html.isEmpty ) {
```

Comment:
You've made changes to how the code determines what makes up a valid article, but haven't updated the tests to reflect this change. I need `ArticleTests` to reflect this change to the logic.

The `matches(_ query: String)` function in this class should also parse the `html` value is a `body` value is not present, as it stands any article with _only_ HTML content does not get included in relevant search results.